### DISCUSSION TITLE
No access to "owner's account"

### DISCUSSION THREAD

**Discussion by mvtslot** [timdorr/tesla-api#451] [General]
Hi, I'm driving a Tesla model 3 via a lease company. The effect is that I do not have access to the Tesla "owner's account". Instead I do have a "user account" for use in the Tesla IOS app. Do you know if the tesla-api development will work with the user account?

Thanks. 

Marco

**Comment by wooter** [comment]
Since the Tesla app uses the same API TeslaMate uses, you will be able to use TeslaMate. Your Tesla app also keeps track of your location and charging, so your TeslaMate instance will also be able to track when you are charging or driving.

**Reply by mvtslot** [reply]
Thanks for your reply. I'll give it a try soon.
